install/kubernetes/helm/istio/charts/pilot/values.yaml

```diff
@@ -14,6 +14,16 @@ sidecar: true
 traceSampling: 1.0
 # if protocol sniffing is enabled. Default to false.
 enableProtocolSniffing: false
+# Automatic protocol detection uses a set of heuristics to
```

Comment:
this entire thing should go into mesh config.. the global section.. not pilot.. search for the connectionTimeout parameter